### DISCUSSION TITLE
What is your favorite Anime? / 好きなアニメを教えてください！

### DISCUSSION THREAD

**Discussion by ghost** [NAIST-SE/AttractingContributors#558] [General]
What is your favorite Anime?
好きなアニメはなんですか？下のリンクからお願いします．
https://github.com/MakotoSato0321/AttractingContributors-Anime

**Comment by HK-TO** [comment]
I think Fullmetal Alchemist: Brotherhood is the best anime.

And please see my repository: https://github.com/HK-TO/AttractingContributors-Cooking
and discussion: #455

**Reply by ghost** [reply]
Thanks!
